src/test/java/edu/cornell/kfs/pdp/batch/fixture/StandardEntryClassConversionFixture.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.pdp.batch.fixture;
+
+import org.kuali.kfs.pdp.businessobject.options.StandardEntryClassValuesFinder;
+import edu.cornell.kfs.coa.businessobject.options.CuCheckingSavingsValuesFinder;
+
+public class StandardEntryClassConversionFixture {
+    
+    public enum ExpectFailureData {
+        NULL_TEST("NULL_TEST", null),
+        BLANK_TEST("BLANK_TEST", ""),
+        WHITESPACE_TEST("WHITESPACE_TEST", "        "),
+        SINGLE_CHAR_TEST("SINGLE_CHAR_TEST", "P"),
+        TWO_CHAR_TEST("TWO_CHAR_TEST", "PD"),
+        INVALID_THREE_CHAR_TEST("INVALID_THREE_CHAR_TEST", "ABC"),
+        INVALID_FOUR_CHAR_TEST("INVALID_FOUR_CHAR_TEST", "PPD2");
+    
+        public final String typeOfTest;
```

Comment:
Your test enums don't appear to be actively using their `typeOfTest` fields, and it seems they're just being set to the names of their respective enum constants anyway. You may want to remove them to reduce clutter.